src/src/DataStructs/Web_StreamingBuffer.h

```diff
@@ -76,8 +80,9 @@ class Web_StreamingBuffer {
 private: 
 
   void sendContentBlocking(String& data);
-  void sendHeaderBlocking(bool          json,
-                          const String& origin = "");
+  void sendHeaderBlocking(bool          allowOriginAll,
+                          const String& content_type,
+                          const String& origin);
```

Comment:
We're loosing the default `= ""` on `origin` here, is that intentional?

Reply:
Yep, it is a private function and only called from one point.
So better not to leave default parameters if they aren't used optionally.